mcda/electre/class_assignment.py

```diff
@@ -0,0 +1,194 @@
+from typing import Any, Dict, Union
+
+from ..core.aliases import NumericValue
+from ..electre.outranking import OutrankingRelation
+
+import pandas as pd
+
+
+def get_relation_type(
+    x: Union[int, str], y: Union[int, str], outranking: pd.DataFrame
+) -> OutrankingRelation:
+    """
+    Assigns type of relation according to credibility
+    :param x: index of row
+    :param y: index of column
+    :param outranking: dataframe containing outranking values
+    :return:
+    """
+    if outranking[x][y] and outranking[y][x]:
+        relation = OutrankingRelation.INDIFF
+    elif outranking[x][y] and not outranking[y][x]:
+        relation = OutrankingRelation.PQ
+    elif not outranking[x][y] and not outranking[y][x]:
+        relation = OutrankingRelation.R
+    else:
+        relation = OutrankingRelation.R
+
+    return relation
```

Comment:
we don't need the `relation` variable, you can simplify the code and type `return` instead of the assignment